apps/meteor/app/oembed/server/server.ts

```diff
@@ -151,10 +188,6 @@ const getUrlMeta = async function (
 		return;
 	}
 
-	if (content.attachments) {
```

Comment:
Is this change really necessary here? I saw your comment `Removed attachments check on...` on the PR description. I believe if this is not a requirement for the PR and the feature, we should not change that now, it would be better to create another PR only with this change. WDYT?